### PR TITLE
Couple of stdlib additions

### PR DESCRIPTION
**Pull request by TheOnlyArtz**
- Reverse an array 
- Query 2D maps by a key and a value
- Get an array of values from map

### CODE REVIEW

File: stdlib/maps.mni [hunk 1]
```diff
@@ -52,14 +60,7 @@ class Map {
     }
 
     show() {
-        let r = "{ ";
```
Comment (Kalekdan) [inline stdlib/maps.mni]: Any reason why this function has been deleted?

Comment (crazywolf132) [reply stdlib/maps.mni]: The += is not yet supported for strings. Only numbers. 

Once this is supported, it will go back.

**Comment by crazywolf132** [comment]
@Kalekdan Yes, it should be placed in the lists test.

**Comment by crazywolf132** [comment]
All looks good. Merging

**Pull request merged**